api-controllers/ServiceFabrikAdminController.js

```diff
@@ -346,25 +346,20 @@ class ServiceFabrikAdminController extends FabrikBaseController {
           logger.warn(`Found deployment '${deployment.name}' without service instance`);
           return false;
         }
-        return eventmesh.apiServerClient.getPlatformContext({
-          resourceGroup: CONST.APISERVER.RESOURCE_GROUPS.DEPLOYMENT,
-          resourceType: CONST.APISERVER.RESOURCE_TYPES.DIRECTOR,
-          resourceId: this.getInstanceId(deployment.name)
-        })
-          .then(context => {
-            const opts = {};
-            opts.context = context;
-            return deployment.directorService
-              .diffManifest(deployment.name, opts)
-              .then(result => _
-                .chain(deployment)
-                .assign(_.pick(result, 'diff', 'manifest'))
-                .get('diff')
-                .size()
-                .gt(0)
-                .value()
-              );
-          });
+        const opts = {};
+        opts.context = {
```

Comment:
@jayeshjk I don't think it will work as other fields like org_guid and space_guid also are part of original context. It will always show diff as `tags.org_guid` and `tags.space_guid` in [manifest](https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/master/operators/bosh-operator/DirectorService.js#L870)